src/services/resource.js

```diff
@@ -81,10 +99,70 @@ module.exports = class resourceHelper {
 
 				const inReviewResources = await resourceQueries.findAll(filter, ['id'])
 
-				const uniqueInReviewResourcesIds = [...new Set(inReviewResources.map((item) => item.id))]
-
+				uniqueInReviewResourcesIds = utils.getUniqueElements(inReviewResources.map((item) => item.id))
+
+				const fetch_resource_ids_from_reviews = await reviewsQueries.findAll(
+					{
+						resource_id: {
+							[Op.in]: uniqueResourceIds,
+						},
+						status: {
+							[Op.in]: [
+								common.REVIEW_STATUS_INPROGRESS,
+								common.REVIEW_STATUS_REJECTED,
```

Comment:
@adithyadinesh0412 why rejected is taking here?

Reply:
as discussed It is being used in the next line for the response and i button we are giving additional custom keys. so added.